src/peft/tuners/xlora/model.py

```diff
@@ -170,8 +170,8 @@ class XLoraModel(BaseTuner):
 
     Example:
         ```py
-        >>> from transformers import AutoModelForCausalLM, AutoConfig
-        >>> from peft import LoraConfig, PeftModel, get_peft_model, prepare_model_for_int8_training
```

Comment:
Sorry I didn't notice earlier: `prepare_model_for_int8_training` is deprecated, but `prepare_model_for_kbit_training` replaces it and can be used here.

Reply:
Ah ok thanks! I updated it to call `prepare_model_for_kbit_training`.